### PR TITLE
IR: Process set eACL notifications

### LINKED ISSUE #505: Approve eACL changes in inner ring
Related to https://github.com/nspcc-dev/neofs-contract/issues/76

Container processor in inner ring should process `SetEACL` method and check container signatures in `Put` and `Delete`.

### PR DESCRIPTION
**Pull request by cthulhu-rider**
Closes #505.

### CODE REVIEW

File: pkg/morph/client/container/wrapper/container.go [hunk 1]
```diff
@@ -17,51 +18,85 @@ var (
...
-// If an empty slice is returned for the requested identifier,
-// storage.ErrNotFound error is returned.
-func (w *Wrapper) Get(cid *container.ID) (*container.Container, error) {
-	if cid == nil {
-		return nil, errNilArgument
+// Returns calculated container identifier and any error
+// encountered that caused the saving to interrupt.
+//
+// If TryNotary is provided, call notary contract.
+func (w *Wrapper) Put(cnr, key, sig []byte) error {
```
Comment (alexvanin) [inline pkg/morph/client/container/wrapper/container.go:53]: PutBinary

File: pkg/morph/client/container/wrapper/container.go [hunk 2]
```diff
@@ -86,25 +121,34 @@ func (w *Wrapper) Get(cid *container.ID) (*container.Container, error) {
...
 // Delete removes the container from NeoFS system
 // through Container contract call.
 //
 // Returns any error encountered that caused
 // the removal to interrupt.
-func (w *Wrapper) Delete(cid *container.ID, signature []byte) error {
-	if cid == nil || len(signature) == 0 {
+//
+// If TryNotary is provided, calls notary contract.
+func (w *Wrapper) Delete(cid, signature []byte) error {
```
Comment (alexvanin) [inline pkg/morph/client/container/wrapper/container.go:143]: DeleteBinary

File: pkg/morph/client/container/wrapper/container.go [hunk 3]
```diff
@@ -17,51 +18,85 @@ var (
...
+func Get(w *Wrapper, cid *container.ID) (*container.Container, error) {
+	return w.Get(cid.ToV2().GetValue())
+}
+
+// Get reads the container from NeoFS system by binary identifier
+// through Container contract call.
+//
+// If an empty slice is returned for the requested identifier,
+// storage.ErrNotFound error is returned.
+func (w *Wrapper) Get(cid []byte) (*container.Container, error) {
```
Comment (alexvanin) [inline pkg/morph/client/container/wrapper/container.go:96]: GetBinary

**Review by alexvanin — APPROVED** [review]

File: pkg/morph/client/container/wrapper/container.go (hunk 1 above)
Comment (cthulhu-rider) [reply pkg/morph/client/container/wrapper/container.go:53]: There are no other `Put...`

Comment (cthulhu-rider) [reply pkg/morph/client/container/wrapper/container.go:53]: No, here we use same naming, the difference only in argument types.

File: pkg/morph/client/container/wrapper/container.go (hunk 2 above)
Comment (cthulhu-rider) [reply pkg/morph/client/container/wrapper/container.go:143]: See comment about Put

File: pkg/morph/client/container/wrapper/container.go (hunk 3 above)
Comment (cthulhu-rider) [reply pkg/morph/client/container/wrapper/container.go:96]: See comment about Put

**Pull request merged**